app/serializers.py

```diff
@@ -0,0 +1,42 @@
+import json
+import xml.etree.ElementTree as ET  # noqa: N817
+from abc import ABC, abstractmethod
+
+
+class Serializer(ABC):
+    @staticmethod
+    @abstractmethod
+    def serialize(title: str, content: str) -> str:
+        pass
+
+
+class JSONSerializer(Serializer):
+    @staticmethod
+    def serialize(title: str, content: str) -> str:
+        return json.dumps({"title": title, "content": content})
+
+
+class XMLSerializer(Serializer):
+    @staticmethod
```

Comment:
Making serialize a static method turns the class into something that behaves more like a utility class (just holding functions), which doesn't align well with the intention behind object-oriented design. Serializer classes should ideally represent instances with behavior (like serialization formats) rather than merely functioning as containers for static utility methods.